python/flow_sdk/shim_airbyte_cdk.py

```diff
@@ -105,8 +105,8 @@ def from_flow_state(state: dict | None, bindings: t.List[CaptureBinding]):
             return (parsed_state, index)
 
         except Exception as e:
-            logger.error("Failed to parse incoming state: ", e)
-            raise e
+            logger.error("Failed to parse incoming state, letting it ride: ", e)
```

Comment:
I switched this back to letting state parse errors pass since I ran into the issue again when upgrading fb marketing tasks, where the old checkpoints had invalid state according to the new model. I say we leave it for the moment, as we're likely to encounter this again in the future.